packages/block-library/src/navigation-menu/index.php

```diff
@@ -97,22 +97,33 @@ function build_navigation_menu_html( $block, $colors ) {
 	$html = '';
 	foreach ( (array) $block['innerBlocks'] as $key => $menu_item ) {
 
+		$opens_in_new_tab = isset( $menu_item['attrs']['opensInNewTab'] ) ?? false;
```

Comment:
`??` will break PHP compatibility unfortunately. We should be able to use `?:` if `$menu_item['attrs']` indeed can be empty.

Reply:
I think `??` is that little magic gem that simply hurts when not having it :))

Reply:
Absolutely. I only noticed it because I ran into the same thing in #18071.